interpreter/cling/include/cling/Interpreter/Interpreter.h

```diff
@@ -517,6 +518,11 @@ namespace cling {
     ///
     std::string lookupFileOrLibrary(llvm::StringRef file);
 
+    ///
+    ///function to replace the environmental variables in a given path.
+    ///
+    void replaceEnvVars(std::string &LibPath);
```

Comment:
Given that the only user is in ClingPragma.cpp, could you make this a local static function instead of declaring it in Interpreter.h?